.github/workflows/test-azureblob.yml

```diff
@@ -0,0 +1,46 @@
+on:
+    push:
+        branches:
+            - master
+            - main
+    pull_request:
+name: "Tests Azure Blob"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                go-version:
+                    - 1.18.x
+                    - 1.19.x
+                platform:
+                    - ubuntu-latest
+                    - windows-latest
+        steps:
+        -   name: Install Azurite
+            run: |
+                docker run -d -p 10000:10000 -e AZURITE_ACCOUNTS="azurite:YXp1cml0ZWtleQo=" mcr.microsoft.com/azure-storage/azurite azurite-blob --blobHost 0.0.0.0 --blobPort 10000

```

Comment:
Any reason this uses a specific account, instead of the default one ?

Reply:
No reasons, just wanted to test with custom credentials. Fixed.